src/sentry/models/sentryapp.py

```diff
@@ -135,6 +144,16 @@ def is_unpublished(self):
     def is_internal(self):
         return self.status == SentryAppStatus.INTERNAL
 
+    @property
+    def datadog_name(self):
```

Comment:
@ceorourke Nit: I actually liked @MeredithAnya suggestion of the name for this function better than mine: `slug_for_metrics`